### DISCUSSION TITLE
Image hashes for covers

### DISCUSSION THREAD

**Discussion by mizaki** [GrandComicsDatabase/gcd-django#615] [Ideas]
Recently, Metron added image hashes to their [DB/API](https://github.com/bpepple/metron/pull/171), [discussion here](https://github.com/bpepple/metron/issues/170). Because you do not have an API (currently), it may be of little interest but having them in the dump could be of use to [some people](https://github.com/mizaki/comictagger-gcd-talker), or yourselves in the future with your own API. 

Is it something you would consider?

**Comment by jochengcd** [comment]
With the distribution of comic images we are always very careful on what we provide. I am not sure that we would provide them with an API even if we would do one. Only if the user of the API acknowledges to be (financially) responsible for any potential troubles we might come into by providing covers via an API. So it would be a sub-API in some sense.

If we would do it, an image hash might by a possibility, or we would provide the data of the last update of the image.

**Reply by mizaki** [reply]
Providing the hash would help get around the image URL issue for identification.

I'm sure you don't want to get into but I'm kind of intrigued. Isn't the (possible) problem not providing a URL to an image (so many APIs already do), but the fact that you host them at all? I would have thought (not an IP lawyer) any liability is already there. If Comic Vine hasn't had any problem? I'm sure you already honour DMCA take down notices too.

**Comment by jochengcd** [comment]
We see a difference between displaying a cover on an individual page for identification of an issue vs. making cover images available in bulk, which an API would allow.

**Comment by mizaki** [comment]
With the API now "in-the-works" (and ComicTagger having better support), is it possible to revisit the possibility of adding cover image hashes?
